src/locale/locales/fr/messages.po

```diff
@@ -4107,17 +3844,13 @@ msgstr "Sélectionnez ce que vous voulez voir (ou ne pas voir), et nous nous occ
 msgid "Select which languages you want your subscribed feeds to include. If none are selected, all languages will be shown."
 msgstr "Sélectionnez les langues que vous souhaitez voir figurer dans les fils d’actu que vous suivez. Si aucune langue n’est sélectionnée, toutes les langues seront affichées."
 
-#: src/view/screens/LanguageSettings.tsx:98
-#~ msgid "Select your app language for the default text to display in the app"
-#~ msgstr "Sélectionnez la langue de votre application à afficher par défaut"
-
 #: src/view/screens/LanguageSettings.tsx:98
 msgid "Select your app language for the default text to display in the app."
-msgstr ""
+msgstr "Sélectionnez la langue de votre application à afficher par défaut."
```

Comment:
I think it's the language that is `your`, not the app; also I add back the `text` part of the sentence to highlight that it's about "static" text, not content.

```suggestion
msgstr "Sélectionnez votre langue par défaut pour les textes de l’application."
```